release-notes/unreleased/IGAPP-1308-remove-ios-image-caching.yml

```diff
@@ -0,0 +1,6 @@
+issue_key: IGAPP-1308
```

Comment:
Not sure if release notes for issues like this makes sense. There is not really anything to test for martin, nothing interesting for users and also not relevant for us devs (since the ticket/PR/commit message give enough info). Our guideline was so far to just add release notes if relevant for users, testers or the other teams.